examples/authsignal-passwordless/pages/api/logout.ts

```diff
@@ -0,0 +1,10 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { serialize } from "cookie";
+import { COOKIE_NAME } from "../../lib";
+
+export default async function logout(_: NextApiRequest, res: NextApiResponse) {
+  const cookie = serialize(COOKIE_NAME, "", { maxAge: -1, path: "/" });
```

Comment:
Should this endpoint only accept a POST request with some added CSRF protection? 🤔